packages/cli/package.json

```diff
@@ -10,13 +10,13 @@
   "dependencies": {
     "@oclif/core": "^2.15.0",
     "@subql/common": "workspace:*",
-    "@subql/common-algorand": "^3.4.0",
-    "@subql/common-concordium": "^3.6.0",
-    "@subql/common-cosmos": "^4.3.0",
-    "@subql/common-ethereum": "^3.8.3",
-    "@subql/common-near": "^3.5.0",
-    "@subql/common-stellar": "^3.5.0",
-    "@subql/common-substrate": "workspace:*",
+    "@subql/types": "workspace:*",
+    "@subql/types-algorand": "^3.3.0",
+    "@subql/types-concordium": "^3.6.0",
+    "@subql/types-cosmos": "^3.4.1-0",
+    "@subql/types-ethereum": "^3.7.1-2",
+    "@subql/types-near": "^3.3.0",
+    "@subql/types-stellar": "^3.3.1",
```

Comment:
I think these still need to be removed. A lot of these types depend on the underlying chain SDK (eg polkadot, ethers) which wont reduce the dependency size